packages/ember-runtime/lib/mixins/array.js

```diff
@@ -638,7 +638,7 @@ const ArrayMixin = Mixin.create(Enumerable, {
 function EachProxy(content) {
   this._content = content;
   this._keys = undefined;
-  this.__ember_meta__ = null;
+  this.__ember_meta__ = undefined;
```

Comment:
I think we can strip it once we drop support for old browsers entirely. For now, keeping the shape stable seems important?